config/deskStructure.ts

```diff
@@ -172,3 +191,54 @@ const hentMapper = (type, delmaler: IDokument[]): IMappe => {
 
   return mapper;
 };
+
+const tomMappe: IMappe = { dokumenter: [], undermapper: {} };
+
+const hentMapperKsBegrunnelse = (type, begrunnelser: IBegrunnelse[]): IMappe => {
```

Comment:
Velger å la behandlingstema og begrunnelsetype bestemme mappestrukturen for ks-begrunnelser enn så lenge.